MainRepository/Program.cs

```diff
@@ -1,7 +1,17 @@
-﻿internal class Program
-{
-    private static void Main(string[] args)
-    {
-        Console.WriteLine("Hello, World!");
-    }
-}
\ No newline at end of file
+﻿// Reduce string
+Console.Write ("Enter the string: ");
+string s = Console.ReadLine ();
+bool reduced;
+do {
+   reduced = false;
+   for (int i = 0; i < s.Length - 1; i++) {
+      if (s[i] == s[i + 1]) {
+         s = s.Substring (0, i) + s.Substring (i + 2);

```

Comment:
Computationally costly. Instead of while() and for(), you could use single loop to solve the same.